connector_alndata/models/res_partner.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2019 Open Source Integrators
+# <https://www.opensourceintegrators.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    partner_type = fields.Selection(
+        [('owner', 'Owner'),
+         ('management_company', 'Management Company'),
+         ('contact', 'Contact')],
+        'Partner Type')
+    lastdate_changed = fields.Datetime('LastDateChanged')
+    address_type = fields.Char('AddressType')
+    rowversion = fields.Char('Row Version')
```

Comment:
@nikitavaghela We don't need to store this information for each record. Only for the whole data set in the system parameters.

Reply:
okay